File: person-management-frontend/types/person.ts
export interface Person {
  personId: number;
  firstName: string;
  lastName: string;
  age: number;
  createdAt: string;
  updatedAt?: string;
}

export interface CreatePersonDto {
  firstName: string;
  lastName: string;
  age: number;
}

export interface UpdatePersonDto {
  firstName: string;
  lastName: string;
  age: number;
}

export interface ApiError {
  message: string;
  errors?: Record<string, string[]>;
}

File: PersonManagement-backend/PersonManagement.Domain/Entities/Person.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace PersonManagement.Domain.Entities
{
    public class Person
    {
        [Key]
        public int PersonId { get; set; }

        [Required]
        [StringLength(50, MinimumLength = 2)]
        public string FirstName { get; set; }

        [Required]
        [StringLength(50, MinimumLength = 2)]
        public string LastName { get; set; }

        [Required]
        [Range(1, 100)]
        public int Age { get; set; }

        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}